packages/backend-app-api/api-report.md

```diff
@@ -121,6 +122,27 @@ export interface DefaultRootHttpRouterOptions {
   indexPath?: string | false;
 }
 
+// @public
```

Comment:
ideally, we could extend the `BackendInitializer` with just the CLI specific start + stopping logic without having to copy everything. I didn't want to touch the `BackendInitializer` internals here though without figuring out a solid path first. Very open to ideas on how we can allow extending the initialization instead of full forking though.